assets/css/webfontkit/stylesheet.css

```diff
@@ -0,0 +1,128 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on April 13, 2017 */
```

Comment:
Put these rules right after your CSS reset in your main stylesheet (update font file paths).

Remove unnecessary font files.